### PR TITLE
AArch64: Implement evaluator for vector fma

### PR DESCRIPTION
**Pull request by Akira1Saitoh**
This commit implements an evaluator for vector fma.

Signed-off-by: Akira Saitoh <saiaki@jp.ibm.com>

### CODE REVIEW

File: compiler/aarch64/codegen/OMRTreeEvaluator.cpp [hunk 1]
```diff
@@ -1577,7 +1577,46 @@ OMR::ARM64::TreeEvaluator::vdsqrtEvaluator(TR::Node *node, TR::CodeGenerator *cg
...
 OMR::ARM64::TreeEvaluator::vfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild = node->getThirdChild();
+
+   TR::Register *thirdReg  = cg->evaluate(thirdChild);
+   TR::Register *secondReg  = cg->evaluate(secondChild);
+   TR::Register *firstReg  = cg->evaluate(firstChild);
```
Comment (knn-k) [inline compiler/aarch64/codegen/OMRTreeEvaluator.cpp:1586]: Is there any reason for evaluating the child nodes in the order from third to first?

Comment (Akira1Saitoh) [reply compiler/aarch64/codegen/OMRTreeEvaluator.cpp:1586]: No. I think we can do it from first to third for simplicity.

**Comment by knn-k** [comment]
Jenkins build aarch64

**Comment by knn-k** [comment]
Jenkins build aarch64

**Review by knn-k — APPROVED** [review]

**Pull request merged**